packages/webview_flutter/webview_flutter_platform_interface/lib/src/platform_ssl_auth_error.dart

```diff
@@ -0,0 +1,40 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+
+import 'types/types.dart';
+
+/// Represents an SSL error with the associated certificate.
+///
+/// The host application must call [cancel] or, contrary to secure web
+/// communication standards, [proceed] to provide the web view's response to the
+/// request. [proceed] should generally only be used in test environments, as
+/// using it in production can expose users to security and privacy risks.
+abstract class PlatformSslAuthError {
+  /// Creates a [PlatformSslAuthError].
+  @protected
+  PlatformSslAuthError({required this.certificate, required this.description});
+
+  /// The certificate associated with this error.
+  final X509Certificate? certificate;
+
+  /// A human-presentable description for a given error.
+  final String description;
+
+  /// Instructs the WebView that encountered the SSL certificate error to ignore
+  /// the error and continue communicating with the server.
+  ///
+  /// **Warning:** When an SSL error occurs, the host application should always
+  /// call [cancel] rather than [proceed] because an invalid SSL certificate
```

Comment:
This was supposed to get the same wording change as above: "**Warning:** This should generally only be called in test environments, as using it in production can expose users to security and privacy risks."

Reply:
Changed it to match: https://github.com/flutter/packages/pull/9150#discussion_r2077704414